test/e2e/api_inheritance/api_inheritance_test.go

```diff
@@ -65,69 +65,62 @@ func TestAPIInheritance(t *testing.T) {
 			t.Errorf("failed to construct apiextensions client for server: %v", err)
 			return
 		}
-		crdAdminClient := apiExtensionsClients.Cluster("admin").ApiextensionsV1().CustomResourceDefinitions()
-
-		// Register CRDs in the admin logical cluster
-		adminClusterCRDs := []metav1.GroupKind{
-			{Group: tenancy.GroupName, Kind: "workspaces"},
-		}
-
-		if err := config.BootstrapCustomResourceDefinitions(ctx, crdAdminClient, adminClusterCRDs); err != nil {
-			t.Errorf("failed to bootstrap CRDs: %v", err)
-			return
-		}
 
 		kcpClients, err := clientset.NewClusterForConfig(cfg)
 		if err != nil {
 			t.Errorf("failed to construct kcp client for server: %v", err)
 			return
 		}
 
-		// For now, API inheritance only supports workspaces in the admin logical cluster, so let's create
-		// a couple there.
-		kcpAdminClient := kcpClients.Cluster("admin")
+		kcpOrganizationClient := kcpClients.Cluster(helper.OrganizationCluster)
```

Comment:
So ultimately I think this is the crux of it - what do we actually want to happen with this feature? If two `Workspace`s can support inheritance at one "level" ... e.g. within an org, one can inherit from the other, we actually want this to be some organizational workspace we're watching. As written, this (poorly named) constant has us watching the root logical cluster where we find organizational workspaces.

In the far future we expect to have N logical clusters on the kcp shard that mirror N organizational workspaces, right? If a shard holds workspaces from N orgs. So we would want to watch all of those for inheritance.

So perhaps we could still do a `*` client here, but the actual logical clusters in which we create `Workspace` objects would be e.g. `admin_org-a`, `admin_org-b` etc? The actual `Workspace`s would be e.g. `org-b_ws-1`...

Reply:
Yes, I guess two organizational workspaces can have inheritance? I think a `*` client is still necessary then, so it would be able to see `Workspace` objects regardless of if they were mirrored from the root logical cluster, or any organizational workspace. It may be easier to grok it all if you do `acme_team-abc` in the test (or that _and_ two org workspaces)